src/client/driver/driver.js

```diff
@@ -379,7 +379,7 @@ export default class Driver {
     _onPrepareBrowserManipulationCommand () {
         this.contextStorage.setItem(this.COMMAND_EXECUTING_FLAG, true);
 
-        prepareBrowserManipulation(this.browserId)
+        prepareBrowserManipulation()
```

Comment:
it seems we can get rid of `this.browserId` now: https://github.com/DevExpress/testcafe/blob/master/src/client/driver/driver.js#L83